### PR TITLE
Add handwritten score and raising-hand models

### PR DESCRIPTION
**Pull request by snosov1**

### CODE REVIEW

**Comment by kchechil** [comment]
@snosov1 index.md update is missing

File: intel_models/person-detection-raisinghand-recognition-0001/description/person-detection-raisinghand-recognition-0001.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+| Min pedestrian height             | 80 pixels (on 1080p)                      |
+| GFlops                            | 7.138                                     |
+| MParams                           | 1.951                                     |
+| Source framework                  | Caffe*                                    |
+
+Average Precision (AP) is defined as an area under the [precision/recall](https://en.wikipedia.org/wiki/Precision_and_recall)
+curve.
+
+## Performance
+Link to [performance table](https://software.intel.com/en-us/openvino-toolkit/benchmarks)
```
Comment (zmaslova) [inline intel_models/person-detection-raisinghand-recognition-0001/description/person-detection-raisinghand-recognition-0001.md]: please remove this line

**Comment by snosov1** [comment]
@zmaslova please, review

File: intel_models/index.md [hunk 2]
```diff
@@ -18,21 +18,22 @@ network to detect objects of the same type better.
...
+| [person-detection-retail-0013](./person-detection-retail-0013/description/person-detection-retail-0013.md)                                                                          | 2.300                | 0.723      |       | X       |          |       |                |
+| [person-detection-action-recognition-0005](./person-detection-action-recognition-0005/description/person-detection-action-recognition-0005.md)                                      | 7.140                | 1.951      |       | X       |          |       |                |
+| [person-detection-action-recognition-teacher-0002](./person-detection-action-recognition-teacher-0002/description/person-detection-action-recognition-teacher-0002.md)              | 7.140                | 1.951      |       | X       |          |       |                |
+| [person-detection-raisinghand-recognition-0001.md](./person-detection-raisinghand-recognition-0001/description/person-detection-raisinghand-recognition-0001.md)                    | 7.140                | 1.951      |       | X       |          |       |                |
```
Comment (zmaslova) [inline intel_models/index.md]: `[person-detection-raisinghand-recognition-0001](.`

**Comment by snosov1** [comment]
@zmaslova Zoya, ok?

**Comment by zmaslova** [comment]
@snosov1 today we will update precommits, and I'll merge after it. looks good

**Comment by zmaslova** [comment]
Jenkins please retry a build

**Comment by zmaslova** [comment]
@snosov1 please rebase

**Comment by snosov1** [comment]
@zmaslova, rebased.

**Pull request merged**